tests/foreman/api/test_classparameters.py

```diff
@@ -1086,13 +1435,26 @@ def test_positive_update_hidden_value_in_parameter(self):
 
         1. The parameter default value is updated.
         2. The 'hidden value' set/displayed as True for that parameter.
-
-        @caseautomation: notautomated
         """
+        old_value = gen_string('alpha')
+        new_value = gen_string('alpha')
+        sc_param = self.sc_params_list.pop()
+        sc_param.override = True
+        sc_param.default_value = old_value
+        sc_param.hidden_value = True
+        sc_param.update(['override', 'default_value', 'hidden_value'])
+        sc_param = sc_param.read()
+        self.assertEqual(getattr(sc_param, 'hidden_value?'), True)
+        self.assertEqual(sc_param.default_value, old_value)
+        sc_param.default_value = new_value
+        sc_param.hidden_value = False
```

Comment:
Remove this not required as you are not updating the parameter hidden_value to false.